Maquetado/index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>HTML & Training</title>
+    <meta name="description" content="Training">
+    <meta name="author" content="Wolox">
+    <link rel="stylesheet" type="text/css" href="application.css"
+</head>
+<body>
+    <h1 class="title">Hello World</h1>
+</body>
+</html>
```

Comment:
Add a empty line at the end of all your files so this simbol do not appear here.